ProgettoMP2018-2019/test/warehouseTest/ShelfTest.java

```diff
@@ -0,0 +1,67 @@
+package warehouseTest;
+
+import static org.junit.Assert.*;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import myTools.Product;
+import warehouse.Shelf;
+
+public class ShelfTest {
+    private Shelf testShelf;
+    private Product productTest;
+
+    @Before
+    public void initFixture() {
+        testShelf = new Shelf(5);
```

Comment:
Io lo crederei vuoto e in base al tipo di test aggiornerai il numero di posti sullo scaffale.